web/elm/tests/DashboardTests.elm

```diff
@@ -1026,12 +1026,16 @@ all =
                         |> Tuple.first
                         |> Common.queryView
                         |> teamHeaderHasNoPill "team"
-            , test "shows pill for first role on team header for team on which user has multiple roles" <|
+            , test
+                ("shows pill for most-privileged role on team header for team"
+                    ++ "on which user has multiple roles"
```

Comment:
Really minor nit, but there should probably be a space between "team" and "on"

Reply:
heck yeah there should. i obviously didn't watch that test fail! (I did, but only before I changed the text, thus making The Diagnostics worse lol)